HWweek6.py

```diff
@@ -0,0 +1,87 @@
+class TreeNode(object):
+    def __init__(self, x):
+        self.val = x
+        self.left = None
+        self.right = None
+
+def isequal(rootS, rootT):
+    if rootS is None and rootT is None:
```

Comment:
Quick optimization tip: because I actually made the same logic comparison here & Carla pointed out that we can actually only just check if `rootT` is`None`. Because if T is null then it will always be a subtree & always return true!